e2e/fragments/addressPostcodeLookup.js

```diff
@@ -0,0 +1,31 @@
+const {I} = inject();
+
+module.exports = {
+
+  fields: {
+    postcodeLookup: 'input[id$="postcodeInput"]',
+    addressList: 'select[id$="addressList"]',
+    buildingAndStreet: {
+      lineOne: 'input[id$="AddressLine1"]',
+      lineTwo: 'input[id$="AddressLine2"]',
+      lineThree: 'input[id$="AddressLine3"]',
+    },
+    town: 'input[id$="PostTown"]',
+    county: 'input[id$="County"]',
+    country: 'input[id$="Country"]',
+    postcode: 'input[id$="PostCode"]',
+  },
+  findAddressButton: 'Find address',
+  cantEnterPostcodeLink: locate('a').withText('I can\'t enter a UK postcode'),
```

Comment:
Would it be possible to to use apostrophe instead  of single quote?  i.e. `. withText('I can’t enter a UK postcode')`

Reply:
sadly not 

` 1) Claim creation @create-claim
       Solicitor creates claim:
     Clickable element "{"type":"xpath","output":null,"strict":true,"locator":{"xpath":".//a[contains(., 'I can’t enter a UK postcode')]"},"value":".//a[contains(., 'I can’t enter a UK postcode')]"}" was not found by text|CSS|XPath
`

Reply:
it's ok. Not a big deal.